CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**3.0.4- 08/20/24**
```

Comment:
This should be v3.0.3. You should include your change in the block below, as we haven't released 3.0.3 yet